responses/authenticate.go

```diff
@@ -7,22 +7,21 @@ import (
 	"github.com/emersion/go-sasl"
 )
 
+type AuthReplyFunc func(reply string)
```

Comment:
Can this return an error? (Even if we don't use it right now)

Reply:
LOL.  The first version of that had returned `error`.  I will put it back.